about.html

```diff
@@ -52,7 +52,7 @@ <h2>Strategy</h2>
 
 <!-- 2nd Section -->
   <div class="first-section">
-    <img src="img/about-working.png" alt="strategy">
+    <img src="img/about-working.png" alt="working">
```

Comment:
for alt on the img tag, please put something another person may understand, screen reader read this tag for those who can't see, and the picture do not load, it will show the alt tag